### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by ViktorOn**
Changes:
- translation improvement

### CODE REVIEW

File: locales/ru_RU/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -104,7 +104,7 @@ msgstr "%1$sУзнайте больше%2$s."
 
 msgctxt "precise_user_location"
 msgid "%sLearn how we keep your location private%s."
-msgstr "%1$sУзнайте, как мы сохраняем конфиденциальность вашего местоположения%2$s."
+msgstr "%1$sКак мы сохраняем конфиденциальность вашего местоположения%2$s."
```
Comment (ViktorOn) [inline locales/ru_RU/LC_MESSAGES/duckduckgo.po:107]: Removed characters to make a sentence one line
image

Comment (ViktorOn) [inline locales/ru_RU/LC_MESSAGES/duckduckgo.po:107]: image

Comment (GioSensation) [reply locales/ru_RU/LC_MESSAGES/duckduckgo.po:107]: Thanks for raising this. I will forward the issue to my colleagues.

**Comment by GioSensation** [comment]
Thanks for your contribution, @ViktorOn! Merging now. 🎉

**Pull request merged**